Application/EdFi.Ods.WebApi/EdFi.Ods.WebApi.csproj

```diff
@@ -23,7 +23,7 @@
 
   <ItemGroup>
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```

Comment:
@semalaiappan I know this is a package that was already assigned and that you are updating the version, this package does not seem to be used by this project anywhere

Reply:
@ericstockdale removed it